src/test/scala/temple/testfiles/simple.temple

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Now you're just trying to increase your lines changed 😉

Reply:
😉